saaj-ri/src/main/java/module-info.java

```diff
@@ -9,7 +9,132 @@
+ *  <td>boolean</td>
+ *  <td><strong>DEPRECATED</strong> If set to {@code "true"}, header fields containing just text such as the {@code Subject}
+ * and {@code Content-Description} header fields, and long parameter values in structured headers such as {@code Content-Type}
+ * will be folded (broken into 76 character lines) when set and unfolded when read. The default is true.</td>
+ * </tr>
+ *
+ * <tr>
+ *  <td><a id="saaj.lazy.contentlength">saaj.lazy.contentlength</a></td>
+ *  <td>boolean</td>
+ *  <td>The {@code saaj.lazy.contentlength} property can be set by applications that do not care
+ * about the {@code Content-Length} property. This allows SAAJ to handle large Payloads. The default is false.</td>
+ * </tr>
+ *
+ * <tr>
+ *  <td><a id="saaj.lazy.mime.optimization">saaj.lazy.mime.optimization</a></td>
+ *  <td>boolean</td>
+ *  <td>Turning this optimization off has limited utility in the absence of MimePull especially when seen
+ * in the context of {@code SOAPMessage.getAttachments()}  and {@code SOAPMessage.getAttachments(...)} APIs.
+ * However it does help a scenario where an intermediate (SOAP Handler) receives a large {@code MimeMessage}
+ * but never needs to look at the Attachments (but only needs to read/manipulate the SOAP Envelope/Body)
+ * before resending the message to an ultimate recipient. The default is true.</td>
+ * </tr>
+ *
+ * <tr>
+ *  <td><a id="saaj.mime.multipart.ignoremissingendboundary">saaj.mime.multipart.ignoremissingendboundary</a></td>
+ *  <td>boolean</td>
+ *  <td>Normally, when parsing a multipart MIME message, a message that is missing the final end boundary line
+ * is not considered an error. The data simply ends at the end of the input. Note that messages of this form
+ * violate the MIME specification. If the property {@code mail.mime.multipart.ignoremissingendboundary} is set
+ * to @{code false}, such messages are considered an error and a {@code MesagingException} will be thrown
+ * when parsing such a message. Note that this feature does not work when one enables {@code saaj.use.mimepull}.
+ * The default is false.</td>
+ * </tr>
+ *
+ * <tr>
+ *  <td><a id="saaj.mime.optimization">saaj.mime.optimization</a></td>
+ *  <td>boolean</td>
+ *  <td>The {@code saaj.mime.optimization} can be used to switch-off Boyer-Moore algorithm for efficient
+ * Mime-Boundary Parsing of a Mime Packaged Message and fallback to the old Jakarta Mail Mime-Boundary parsing
+ * implementation when set to {@code false}. The default is true.</td>
+ * </tr>
+ *
+ * <tr>
+ *  <td><a id="saaj.mime.soapBodyPartSizeLimit">saaj.mime.soapBodyPartSizeLimit</a></td>
+ *  <td>integer</td>
+ *  <td>The {@code saaj.mime.soapBodyPartSizeLimit} property is used to specify the max amount in bytes a SOAP body part
+ * is allowed to contain. This property can (should) be used to protect against maliciously crafted requests which
+ * can take down the server with {@code OutOfMemoryError}. The default is true.</td>
```

Comment:
I think the default value here should probably be something indicating "unlimited" if not specified? Maybe something like this?
```suggestion
 * can take down the server with {@code OutOfMemoryError}. The default is to not enforce any limit.</td>
```

Reply:
good catch, should be fixed